app/src/androidTest/java/ch/epfl/sweng/favors/utils/UtilsTest.java

```diff
@@ -219,4 +224,17 @@ public void CreateUtils(){
         new Utils();
     }
 
+    @Test
```

Comment:
Not the right  class for this. Create a package with this name `app/src/androidTest/java/ch/epfl/sweng/favors/utils/app/src/androidTest/java/ch/epfl/sweng/favors/utils/email`
You will not have any issues with accessing package private stuff from there. So you can revert the change of access of the retrofit client back to how it was.